src/ansys/dpf/composites/layup_info/_layup_info.py

```diff
@@ -415,6 +424,15 @@ def get_element_info(self, element_id: int) -> Optional[ElementInfo]:
             assert layer_data[0] + 1 == len(layer_data), "Invalid size of layer data"
             n_layers = layer_data[0]
             is_layered = True
+        elif self.solver_material_to_dpf_id:
```

Comment:
Enhance the ElementInfo by extracting the mapdl material ID from the RST and convert it into the DPF material ID.

Reply:
This function is quite performance sensitive. Could you test the performance of the new branch for a reasonably large model.